explainer.md

```diff
@@ -322,6 +321,26 @@ cancelable.
     that continues allowing a permission on a
     [previously granted](#ui-when-the-permission-is-already-granted) type of
     UI).
+-   `onvalidationstatuschange` - raised when the PEPC switches from being
+    "valid" to "invalid". The PEPC is considered "valid" when the user agent
+    trusts the integrity of the signal if the user were to click to click on it,
+    and "invalid" otherwise. There are many reasons for which an element can
+    become "invalid" as detailed in the [Security](#security) section, but to
+    enumerate a few: element style is invalid, element is covered, element has
+    recently moved, element has changed size, element is not fully visible in
+    the viewport, etc. A new `Event` subclass will be added,
+    `ValidationStatusChangeEvent`, which contains two extra fields:
+    - `boolean isValid` - indicates whether the status has transitioned to
+      "valid" or not.
+    - `Reason? reason` - where `Reason` is an enum indicating the reason why
```

Comment:
We have received feedback from developers trying our prototype that they want to be able to react in real-time if the PEPC is invalid for some reason, in order to replace it with their status quo approach, to prevent breaking a critical user journey.

I might be wrong but I don't think console logs can be intercepted via JS.

Reply:
right, but the reasons are all developer driven/caused: they shouldn't get into a situation in the first place (some they might not be able to recover from). A lot of these reasons also won't be interoperable. All that really matters is that it's been invalidated for some reason (which may show up in the developer console).

Reply:
Agreed, but it might be accidental or unintended. Being able to catch it and do something about it (even if it's just simply recording metrics) seems like a useful feature. 

Also the validation breakage might be in a scenario that happens somewhat rarely, or on specific screen sizes or has other prerequisites making it possible that it won't be caught during testing.